ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -734,6 +734,36 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
     /// Text that appears as the third line of text
     @objc public var footer: String { return footerLabel.text ?? "" }
 
+    @objc public var attributedTitle: NSAttributedString? {
```

Comment:
should have documentation on these new public api. also this is clear that attibutedstring dictionary will triump any token updates right?

Reply:
Yes, the `updateTextColors()` method will check the boolean value which is set in the attributedString property before making any token decisions